prow/config.yaml

```diff
@@ -1902,6 +1902,45 @@ periodics:
       hostPath:
         path: /mnt/disks/ssd0
 
+- name: ci-perf-tests-kubemark-100-benchmark
+  interval: 2h
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170515-c7116fb4
+      args:
+      - "--repo=k8s.io/perf-tests=master"
+      - "--root=/go/src"
+      - "--timeout=10"
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
+        value: prow
+      - name:  JENKINS_GCE_SSH_PRIVATE_KEY_FILE
```

Comment:
Sorry, just copied it from another example. Thought it was expected by bootstrap.py (seeing this line https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L668)
However seems like neither the execute scenario nor my binary use it. Removed it now.

We still require `GOOGLE_APPLICATION_CREDENTIALS` and `USER` for uploading logs I guess. But I'm not sure about the cache-ssd volume. Seems like a place for git-cache, but when exactly do we need it (while using --bare/--clean/--git-cache flags?)